command/testdata/show-json/modules/output.json

```diff
@@ -274,8 +274,8 @@
             }
         },
         "provider_config": {
-            "module_test_foo:test": {
-                "module_address": "module_test_foo",
+            "module.module_test_foo:test": {
```

Comment:
This is a change in how `show -json` output will look, as a result of upstream changes to the `String()` method on `Module`

Reply:
This seems to be the only point that was using the module path with the `module.` prefix. I agree it's noisier, but if we can break this on the 0.13 release it would at least serve to make everything consistent.

Reply:
I think this is a great change; explicit is better! We will need to make sure the TFC and sentinel teams are aware of this change, in case it impacts them.